generators/entity/index.js

```diff
@@ -244,6 +245,11 @@ module.exports = EntityGenerator.extend({
         this.searchEngine = this.fileData.searchEngine || this.searchEngine;
         this.javadoc = this.fileData.javadoc;
         this.entityTableName = this.fileData.entityTableName;
+        if (this.databaseType === 'sql') {
```

Comment:
can you move this into a method in `generator-base-private` so that its can be reused below?